README.md

```diff
@@ -1,2 +1,10 @@
-# aws-consumer-service
-Consumer Service for Screwdriver AWS Integration
+# Screwdriver AWS Consumer Service
+
+> Consumer-Service Package for Screwdriver AWs Integration
+
+This is package for the Screwdriver continuous delivery solution that receives and processes build messages.
```

Comment:
can you elaborate a bit more. Call out the executor with which this will work with.